server/src/lib.rs

```diff
@@ -73,6 +73,16 @@ pub fn is_email_regex(test: &str) -> bool {
   EMAIL_REGEX.is_match(test)
 }
 
+pub fn is_image_content_type(test: &str) -> bool {
+  match isahc::get(test) {
+    Ok(res) => match res.headers().get("Content-Type") {
+      Some(header) => header.to_str().unwrap_or("not_an_img").contains("image"),
```

Comment:
It would be better to check for `.startsWith("image/")`. Or even add a whitelist of image mimetypes.

Reply:
I tried to find a startsWith, but couldn't find one for rust strings. It just seemed simpler to use contains.

Reply:
I think it also might make sense to change this from a bool to just an Ok, Error.

Reply:
I just had to merge this quickly and deploy bc every link upload was caching the entire html page in pictshare.

Reply:
nm I found the startswith. I'm gonna fix this now.